backend/api/models.py

```
@@ -1,120 +1,83 @@
-        (COMPANY, 'Company'),
-        (USER, 'User'),
-    ]
-
-    name = models.CharField(max_length=50, unique=True, choices=ROLE_CHOICES)
-
-    def __str__(self):
-        return self.name
+class CustomUserManager(BaseUserManager):
+    def create_user(self, email, firebase_uid, password=None, **extra_fields):
+        if not email:
+            raise ValueError('Users must have an email address')
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)